plugins/webhooks/index.js

```diff
@@ -4,11 +4,14 @@ const boom = require('boom');
 const joi = require('joi');
 const winston = require('winston');
 const workflowParser = require('screwdriver-workflow-parser');
+const schema = require('screwdriver-data-schema');
 
 const ANNOT_NS = 'screwdriver.cd';
 const ANNOT_CHAIN_PR = `${ANNOT_NS}/chainPR`;
 const ANNOT_RESTRICT_PR = `${ANNOT_NS}/restrictPR`;
-const CHECKOUT_URL_SCHEMA = require('screwdriver-data-schema').config.regex.CHECKOUT_URL;
+const EXTRA_TRIGGERS = schema.config.regex.EXTRA_TRIGGERS;
+const EXTRA_TRIGGERS_REGEXP = new RegExp(EXTRA_TRIGGERS);
```

Comment:
`schema.config.regex.EXTRA_TRIGGERS` is already a regex object.
This line's  instantiation is not necessary?

Reply:
Thanks! I fixed.